app/src/lib/stats/stats-store.ts

```diff
@@ -24,6 +26,17 @@ const StatsOptOutKey = 'stats-opt-out'
 /** Have we successfully sent the stats opt-in? */
 const HasSentOptInPingKey = 'has-sent-stats-opt-in-ping'
 
+const WelcomeWizardInitiatedAtKey = 'welcome-wizard-initiated-at'
```

Comment:
I feel like these keys can be defined as an enum, which has a benefit of giving of addition type safety in the `timeTo` and `getLocalStorageTimestamp` functions.

If you built this to be as generic as possible because we will need to keep track of stepped timing info, then feel free to resolve as what's here is fine for that scenario.

Reply:
Given that we want to get a release out with this sooner rather than later let's punt on this discussion for now and land this PR without such a change.